src/item.cpp

```diff
@@ -3023,9 +3023,9 @@ void item::book_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
     }
     if( book.skill ) {
         const SkillLevel &skill = you.get_skill_level_object( book.skill );
-        if( skill.can_train() && parts->test( iteminfo_parts::BOOK_SKILLRANGE_MAX ) ) {
+        if( parts->test( iteminfo_parts::BOOK_SKILLRANGE_MAX ) ) {
             const std::string skill_name = book.skill->name();
-            std::string fmt = string_format( _( "Can bring your <info>%s skill to</info> "
+            std::string fmt = string_format( _( "Can bring <info>%s skill to</info> "
                                                 "<num>." ), skill_name );
             info.emplace_back( "BOOK", "", fmt, iteminfo::no_flags, book.level );
```

Comment:
```suggestion
            const auto fmt = string_format( _( "Can bring <info>%s skill to</info> "
                                                "<num>." ), skill_name );
            info.emplace_back( "BOOK", "", skill.can_train() ? fmt : colorize( fmt, c_brown ), iteminfo::no_flags, book.level );
```

how about showing whether the skill can be trained?